src/mach.rs

```diff
@@ -440,11 +478,11 @@ impl<'a> Mach<'a> {
         // marshall the sections into something we can actually write
         let mut raw_sections = Cursor::new(Vec::<u8>::new());
         let mut relocation_offset = relocation_offset_start;
-        let mut section_offset = first_section_offset;
+        let mut segment_relative_offset = first_section_offset;
```

Comment:
This is annoying; accidentally changed this, should ignore. too tired to fix now though.